account_mass_reconcile/models/advanced_reconciliation.py

```diff
@@ -1,41 +1,27 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Guewen Baconnier
-#    Copyright 2012 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
-from openerp.osv import orm
-
-
-class easy_reconcile_advanced_ref(orm.TransientModel):
-
-    _name = 'easy.reconcile.advanced.ref'
-    _inherit = 'easy.reconcile.advanced'
-
-    def _skip_line(self, cr, uid, rec, move_line, context=None):
+# © 2012-2016 Camptocamp SA (Guewen Baconnier, Damien Crier, Matthieu Dietrich)
+# © 2010 Sébastien Beau
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
```

Comment:
Just repeating the conventions:

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

from openerp import api, fields, models  # alphabetically ordered